### ISSUE TITLE
Re-generate .cat file

### ISSUE THREAD

**Issue by dennyamarojr** [CHEF-KOCH/nVidia-modded-Inf#27]
Hi Chef,

I'm here again for a little help, I'm trying to install the latest drivers for my graphics card NVIDIA GT 630, but I can't re-generate the .cat and them make the process you give instructions to install the certificate, could you help me please?

**Comment by CHEF-KOCH** [comment]
Not needed, you can [disable Driver Signature Enforcement](https://windowsreport.com/driver-signature-enforcement-windows-10/), install the driver, reboot and re-enable it (reboot).

**Comment by dennyamarojr** [comment]
Thanks for the answer CHEF.

**Comment by CHEF-KOCH** [comment]
Official Method is btw [here](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/using-makecat-to-create-a-catalog-file), also not that hard to follow since MS provides an example.
